addons/event_sale/report/event_event_templates.xml

```diff
@@ -1,24 +1,22 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
-
-    <template id="event_registration_report_template_badge" inherit_id="event.event_registration_report_template_badge">
-        <xpath expr="//div[@id='o_event_name']" position="inside">
-            <div t-if="o.event_ticket_id" class="col-12 text-center" style="padding:0px;">
-                <div style="background: lightgrey; height: 65px;" class="mt16 text-center">
-                    <h3><span t-field="o.event_ticket_id"/></h3>
-                </div>
+    <template id="event_report_template_full_page_ticket_inherit_sale" inherit_id="event.event_report_template_full_page_ticket">
+        <xpath expr="//div[hasclass('o_event_full_page_ticket_side_info_booked_by')]" position="before">
+            <div t-if="attendee and attendee.sale_order_id" class="mb-2">
+                <div class="o_event_full_page_ticket_font_faded o_event_full_page_ticket_small_caps font-weight-bold">Order Ref</div>
+                <div class="o_event_full_page_ticket_small" t-field="attendee.sale_order_id"/>
             </div>
         </xpath>
-    </template>
-
-    <template id="event_event_report_template_badge" inherit_id="event.event_event_report_template_badge">
-        <xpath expr="//div[@id='o_event_attendee_name']" position="inside">
-            <div t-if="bool(len(event.event_ticket_ids))" class="col-12" style="padding: 0px;" t-ignore="true">
-                <div style="background: lightgrey; height: 65px;" class="mt16 text-center">
-                    <h3>Ticket Type</h3>
-                </div>
+        <xpath expr="//div[hasclass('o_event_full_page_ticket_side_info')]" position="inside">
+            <div t-if="attendee and attendee.sale_order_id" class="mb-2">
+                <div class="o_event_full_page_ticket_font_faded o_event_full_page_ticket_small_caps font-weight-bold">Order Date</div>
+                <div class="o_event_full_page_ticket_small" t-esc="attendee.sale_order_id.date_order.date()"/>
+            </div>
+            <div t-if="attendee and attendee.event_ticket_id.price">
+                <div class="o_event_full_page_ticket_font_faded o_event_full_page_ticket_small_caps font-weight-bold">Price</div>
```

Comment:
Not sure, but shoudln't we use what is defined on so line as prive may have been updated in so itself ? or ballek ?